javav2/example_code/migrationhub/Readme.md

```diff
@@ -0,0 +1 @@
+This is migrationhub readme
```

Comment:
Am I looking at this wrong, or is the README effectively blank?

Reply:
Readme in new check in a place holder - will update